assets/css/style.css

```diff
@@ -1,276 +1,428 @@
-.header-container {
+body {
+  background-color: var(--white);
```

Comment:
- [ ] [optional] Kindly change the background color of the body to ``` #E5E5E5 ``` so your design can be like the Figma one

![Screenshot_20220506_171133](https://user-images.githubusercontent.com/25227022/167180150-fc977173-68f6-46e9-aaa0-cd1722a19189.png)